grammar.js

```diff
@@ -421,13 +431,19 @@ module.exports = grammar({
       ))),
       seq(":'", $._single_quoted_continuation),
       seq(':"', $._double_quoted_continuation),
+      ':"#"', // TODO: Remove this hack
       seq('%s', choice(
-        $._uninterpolated_paren
+        $._uninterpolated_angle,
+        $._uninterpolated_bracket,
+        $._uninterpolated_paren,
+        $._uninterpolated_brace
       ))
     ),
 
-    integer: $ => (/0b[01](_?[01])*|0[oO]?[0-7](_?[0-7])*|(0d)?\d(_?\d)*|0x[0-9a-fA-F](_?[0-9a-fA-F])*/),
-    float: $ => (/\d(_?\d)*\.\d(_?\d)*([eE]\d(_?\d)*)?/),
+    integer: $ => (integerPattern),
+    // TODO: When we can backtrack, redefine float as regex instead of seq.
+    // float: $ => (/\d(_?\d)*\.\d(_?\d)*([eE]\d(_?\d)*)?/),
+    float: $ => seq(integerPattern, '.' ,/\d(_?\d)*([eE]\d(_?\d)*)?/),
```

Comment:
Is there an issue with leaving it this way? Or maybe, do we gain anything from redefining it as a `regex` as opposed to a `seq`?